### PR TITLE
updated monday

### PR DESCRIPTION
**Pull request by Mike-Ebert**

### CODE REVIEW

File: 1906-11-12.xml [hunk 1]
```diff
@@ -1842,12 +1843,414 @@
...
+                    Manzini and the Marquis Medici.</p>
+                 </div>
+                 <div type="item"> <head>Steamer Movements</head>
+                    <p>The Messageries Maritimes S.S. Congo, from Merseilles, passed the Straits of Messina on Saturday 
+                    and it is due here to-morrow morning. The Moss liner Ramesses arrived this morning from Liverpool via
+                    Gibraltar and Malta with passengers, mails, and general cargo. The Moss liner Khephren arrived this 
+                    morning from Barry with coal. The Ellerman liner Brittania arrived this morning from Liverpool and Malta
+                    with passengers and a general cargo. The Ellerman liner city of Khios arrived this morning from Constantinople.
+                    The Ellerman liners Arabia and Egyptian will sail to-morrow for Manchester and Liverpool respectively.</p>
+                 </div>
```
Comment (whanley) [inline 1906-11-12.xml]: add another `</div>` tag to close the local and general section

File: 1906-11-12.xml [hunk 2]
```diff
@@ -1842,12 +1843,414 @@
...
+                    <p>Such occurances as that at Portsmouth on Sunday evening are, fortunately, infrequent . we can 
+                        recall no parallel in the recent history of the British Navy. Probably all who were Concerned 
+                        in the emeute are already heartily _ ashamed of their action. In a country which is democratic
+                        in its institutions and sentiments, the task of commanding large bodies of men is one of extreme
+                        difficulty. It becomes more and more essential that officers of the fighting Services should exhibit
+                        facility for command which is baaed on personal respect rather than on the power of punishment which
+                        is at hand, to enforce an order in the last resort. Fortunately the Fleet has never remained a larger
+                        body of officers endowed with this priceless qualification.
+                    </p>
+                </div>
```
Comment (whanley) [inline 1906-11-12.xml]: you need a `</div>` here to close the page.

File: 1906-11-12.xml [hunk 3]
```diff
@@ -1842,12 +1843,414 @@
...
+                    document Lord Cromer gave special attention to the matter, and put himself in 
+                    communication with the Italian Ambassador in London, with a view to preventing further
+                    such exhibitions in the name of Egypt. The results of his Lordship's communication
+                    to the Italian Ambassador were that the Italian Government promised to prohibit any exhibition
+                    to be labelled "Egyptian" without the consent of the Egyptian Government. Lord Cromer,
+                    on the other hand, promised that the Egyptian Government would hence-forth undertake the 
+                    organisation of the Egyptian section in any international show.</p>
+                 </div>
+                 <div type="item"> <head>KINGS BIRTHDAY</head>
+                 </div>
```
Comment (whanley) [inline 1906-11-12.xml]: this tag should come at the end of the article.

File: 1906-11-12.xml [hunk 4]
```diff
@@ -1842,12 +1843,414 @@
...
+                    to resume his work of reform in the University.</p>
+                 </div>
+                 <div type="item"><head>OFFICIAL RESIGNATIONS</head>
+                    <p>At the last meeting of the Counsil of Ministers the following resignations were accepted: Yacoub Pasha Artin,
+                    under Secretary of State for public Instruction, Hussein Bey Ritz, representative of the Parquet at the Alexandria
+                    Mixed Tribunal, and Doctor Mohamed Bey Shukri, professor of medicine at the Kasre-Aini school of medicine.
+                    Yasin Bey Mustapha, Mudir of Keneh,was retired on pension. His successor has not yet been appointed.</p>
+                 </div>
+                 <div type="item"> <head>THE EGYPTIAN UNIVERSITY</head>
+                 </div>
```
Comment (whanley) [inline 1906-11-12.xml]: this tag should come at the end of the article

File: 1906-11-12.xml [hunk 5]
```diff
@@ -1842,12 +1843,414 @@
...
+                    Actions de Capital (net)... ... 50.232f.
+                    Actions de Jouissance (gross)... ... 49.50f.
+                    Actions de Jouissance (net)... ... 38.584f.
+                    Founders Shares (gross)... ... 23.943f. 
+                    Founders Shares (net)... ... 21.937f.
+                    Coupons will be discounted at the company's offices at the Bank of France Rate of Discount.
+                    </p>
+                 </div>
+                 <div type="item"> <head>CAPTAIN WALDBERG'S CAREER</head>
+                 </div>
```
Comment (whanley) [inline 1906-11-12.xml]: this tag goes at end of article

File: 1906-11-12.xml [hunk 6]
```diff
@@ -1842,12 +1843,414 @@
...
+                    enough fight. The last and greatest event of the evening was the fight to a finish between Van de Berg, the Dutchman, and Schackman, the "terrible" German. Scackmann's method did not at all appeal to the audience, nor, we must confess, to ourselves. If he did 
+                    not actually foul repeatedly, he was at any rate very near, and we think he should have been severely cautioned at the first sign of offending. Anyhow, he won after some 35 minutes. Van de Berg made a splendid fight of it, and time after time recovered when defeat seemed 
+                    inevitable. He showed very great strength of arm, and more than once prevented the German from performing his favorite trick of turning over and on to his feet; but the strength and weight of the other was too much for him, and he seemed quite exhausted at the finish.
+                    </p>
+                    <div type="advert">
```
Comment (whanley) [inline 1906-11-12.xml]: insert `</div>` to close previous item

File: 1906-11-12.xml [hunk 7]
```diff
@@ -1842,12 +1843,414 @@
...
+                            
+                            Per S.S. Rameses, arrived this morning from Liverpool:
+                            
+                            Mrs. Allen, Mr. K. P. Birley, Mr, F. R. Beavan, Rev. W. T. and Mrs. Float, Mrs. Gardner, Miss Sveois. Mr. R. J. Jones, MLe. Mecdcnsld, Mr. F. R Middleton, Mrs5 Parvis, Miss Parvis, Mrs. Rodocanachi, Mrs. Sedgewick, Mre. Snmnismondo, Mr. Ed. Khertison, Mr. L.
+                            this
+                            Mioheti, Mr. L. Salvadosi, Mr. C. Plat, Mr. A. Mr. Ed Vierbucher, Mr. Pifsud, Mr. P Vasallo, Mrs. Nahum, Mr. and Mrs. Tong, Mr. Hardy, Mr. Sant, Mr. C Hoon, Mr. and Mrs. L Johnson, Mrs. Isherwood, Mrs. Jones, Miss A. Jones, Mr. and Mrs. Spins, Mr. Briffs, Mrs. Pace,
+                            Mrs. Pace, Mr. Bossina, Mr. Fralliet, Mr. V Citroni and 15 deck passengers.</p>
+                        <div type="advert">
```
Comment (whanley) [inline 1906-11-12.xml]: add `</div>` tag to close previous section (arrivals)

**Comment by Mike-Ebert** [comment]
I updated deliverable 2

**Comment by Mike-Ebert** [comment]
I fixed the changes for deliverable 2. This is also my first version of deliverable 3.

File: 1906-11-12.xml [hunk 8]
```diff
@@ -56,1798 +60,4978 @@
...
+                                <cell><date when="1906-10-27">October 27</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-06">November 6th</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-10">November 10</date> ,,</cell>
+                            </row>
+                            <row>
+
```
Comment (whanley) [inline 1906-11-12.xml]: add a `</div>`

Comment (whanley) [inline 1906-11-12.xml]: remove the section division--the items that follow are independent.

Comment (whanley) [inline 1906-11-12.xml]: this and following items are part of the item above (on Capt Waldberg), and shouldn't be split into separate divs.

Comment (whanley) [inline 1906-11-12.xml]: there's a template for this

Comment (whanley) [inline 1906-11-12.xml]: split div

Comment (whanley) [inline 1906-11-12.xml]: there's a template for this

**Comment by whanley** [comment]
Generally good, but some cleanup needed, including (but not confined to):

Line 1901--you need another `</div>` to close the weather section
Line 2090--you need a `</div>` to close the section
Line 2289 and after: make the whole Capt Walberg article into one big div (to 2432)
2532, 2587--add element attribute
2677 -- correct OCR text in this div
2791 -- use template--also for following divs

Page 5:
`<hi xml:id="bookmark1">` should be `<head>` or something like it.
Later on in the page, there are some articles combined in a single div that should be separate

**Comment by whanley** [comment]
Deliverable 2 complete--good job.
Some updating left for deliverable 3. Telegrams section, for example.
Thanks for starting deliverable 4.

**Comment by whanley** [comment]
Monday issue
- day in header is wrong: it's Monday, not Thursday
- shipping movements not fixed
- calendar of coming events, marche de minet el bassal still do not match original

Tuesday, Wednesday
- update contents of templates
- lots of other details to take care of

**Comment by whanley** [comment]
Monday is complete. Tuesday and Wednesday still need work.

**Pull request merged**